skypy/halos/tests/test_colossus.py

```diff
@@ -105,3 +105,53 @@ def test_colossus_mf():
     assert np.all(z_halo <= np.max(z))
     assert np.all(m_halo >= m_min)
     assert np.all(m_halo <= m_max)
+
+
+@pytest.mark.skipif(not HAS_COLOSSUS, reason='test requires colossus')
+@pytest.mark.flaky
+def test_colossus_concentration():
+    from skypy.halos._colossus import concentration
+    from astropy.cosmology import Planck15
+
```

Comment:
We could add a comment here, briefly explaining what this test is doing.
For example:

```python
    # Check concentration array matches mass length
```
or similar